go/utils/helper.go

```diff
@@ -0,0 +1,45 @@
+package utils
```

Comment:
I see. I am less worries about package name conflict, more worried about if we have a single utils package, everything gets into that package, and the different code for different purpose may disturb each other. If we put utils of different purpose under different packages under util folder, it will not happen. I believe k8s do it as well: https://github.com/kubernetes/kubernetes/tree/master/pkg/util different utils are under different packages under the util folder.